doc/RELEASE_PROCESS.rst

```diff
@@ -4,11 +4,7 @@ Release Process for rst2pdf
 
 This is an outline of what needs to be done in order to release rst2pdf.
 
-#. Install dependencies that you'll need
-   ::
-
-      $ pip install build twine
```

Comment:
No need for additional disparate tools as `uv build` replaces `python -m build` and `uv publish` replaces `twine`